lib/utils/device/deploy.ts

```diff
@@ -301,6 +323,6 @@ async function inspectBuildResults(images: LocalImage[]): Promise<void> {
 	});
 
 	if (failures.length > 0) {
-		exitWithExpectedError(new LocalPushErrors.BuildError(failures));
+		exitWithExpectedError(new LocalPushErrors.BuildError(failures).toString());
```

Comment:
Bug fix. Without this change, the message printed was a rather unelaborated "Build error":
```
[Build]   [main] Step 1/5 : FROM 192.168.2.1:5000/busybox
Build error
```
After this change it is a bit better:
```
[Build]   [main] Step 1/5 : FROM 192.168.2.1:5000/busybox
Some services failed to build:
	main: Get https://192.168.2.1:5000/v2/: http: server gave HTTP response to HTTPS client
```